pkg/frameworks/tensorflow/tensorflow_serving.go

```diff
@@ -28,16 +28,22 @@ const (
 	TensorflowServingImageName  = "tensorflow/serving"
 )
 
-func CreateTensorflowContainer(modelName string, tfSpec *v1alpha1.TensorflowSpec) *v1.Container {
+type TensorflowSpec v1alpha1.TensorflowSpec
+
+func (t *TensorflowSpec) CreateModelServingContainer(modelName string) *v1.Container {
 	//TODO(@yuzisun) add configmap for image, default resources, readiness/liveness probe
 	return &v1.Container{
-		Image:   TensorflowServingImageName + ":" + tfSpec.RuntimeVersion,
+		Image:   TensorflowServingImageName + ":" + t.RuntimeVersion,
 		Command: []string{TensorflowEntrypointCommand},
 		Args: []string{
 			"--port=" + TensorflowServingGRPCPort,
 			"--rest_api_port=" + TensorflowServingRestPort,
 			"--model_name=" + modelName,
-			"--model_base_path=" + tfSpec.ModelURI,
+			"--model_base_path=" + t.ModelURI,
 		},
 	}
 }
+
+func (t *TensorflowSpec) Validate() error {
```

Comment:
Can you cut an issue referring to this?

Reply:
https://github.com/kubeflow/kfserving/issues/84